internal/gofumpt.go

```diff
@@ -516,6 +528,88 @@ func (f *fumpter) joinStdImports(d *ast.GenDecl) {
 	}
 }
 
+// mergeAdjacentFields mutates fieldList to merge adjacent fields if possible
+// and returns fieldList.
+func (f *fumpter) mergeAdjacentFields(fieldList *ast.FieldList) *ast.FieldList {
```

Comment:
nit: this func only uses `fieldList.List`, so just take and return `[]*ast.Field`

Reply:
Done.